packages/gatsby-plugin-sitemap/src/internals.js

```diff
@@ -59,7 +60,7 @@ export const defaultOptions = {
   serialize: ({ site, allSitePage }) =>
     allSitePage.edges.map(edge => {
       return {
-        url: site.siteMetadata.siteUrl + edge.node.path,
+        url: path.join(site.siteMetadata.siteUrl, edge.node.path),
```

Comment:
This might yield wierd results on windows where path separator is `\`

Reply:
Damn! You're right. Didn't think of the windows part. In that case, my change doesn't make sense ofc. Sorry for the noise 😬